packages/react-card/assets/word_logo.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="22" height="23" viewBox="0 0 22 23" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
you'll need to make sure these aren't released along with our published packages. I would suggest you update the migration generator to update the generated `npmignore` for either:
* assets folder -> could be a bit too opinionated and specific to one package
* all svgs -> I don't think we should be shipping any svgs, but I'm not 100% sure on this

Reply:
Should I just use `@fluentui/example-data` instead or is this a v8 thing only?

Reply:
I think it's a v8 thing, but I don't think including it as a dev dep should have too many consequences. @ecraig12345 any reason we can't use that package as a dev dep for stories ?